tests/e2e/common-operator-integ-suite.sh

```diff
@@ -74,6 +79,22 @@ parse_flags() {
   else
     echo "Running on kind"
   fi
+
+  if [ "${SKIP_BUILD}" == "true" ]; then
+    echo "Skipping build"
+  fi
+
+  if [ "${SKIP_DEPLOY}" == "true" ]; then
+    echo "Skipping deploy"
+  fi
+
+  if [ "${OLM}" == "true" ]; then
+    echo "OLM deployment enabled"
+    if [ "${OCP}" == "true" ]; then
+      echo "OLM deploy test is being skipped on OCP due errors with the workaround to avoid the certificate issue"
```

Comment:
What about the following sentence?

```suggestion
      echo "Skipping operator deployment using OLM on OCP clusters due to certificate issues with the internal registry."
```